addons/mail/static/tests/chat_window/chat_window.test.js

```diff
@@ -788,17 +788,17 @@ test("folded chat window should hide member-list and settings buttons", async ()
     await contains(".o-dropdown-item", { text: "Call Settings" });
 });
 
-test("Chat window in mobile are not foldable", async () => {
+test("chat window: fold (mobile)", async () => {
     const pyEnv = await startServer();
     const channelId = pyEnv["discuss.channel"].create({});
     patchUiSize({ size: SIZES.SM });
-    setupChatHub({ opened: [channelId] });
     await start();
     await openDiscuss(channelId);
-    await contains(".o-mail-ChatWindow");
-    await contains(".o-mail-ChatWindow-header.cursor-pointer", { count: 0 });
-    await click(".o-mail-ChatWindow-header");
-    await contains(".o-mail-Thread"); // content => non-folded
+    await contains(".o-mail-ChatWindow .o-mail-Thread");
+    assertChatHub({ opened: [channelId] });
+    await click(".o-mail-ChatWindow-command[title='Fold']");
+    await contains(".o-mail-ChatWindow .o-mail-Thread", { count: 0 });
+    assertChatHub({ folded: [channelId] });
```

Comment:
I'm not fond of assertion in local storage, this is very technical and I prefer it to be used by specific technical tests.

Functional tests like this one should try its best to assert on what's on UI, so showing of chat bubble / chat window